sdk/search/search-documents/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 11.3.0 (2021-09-??)
+
+### New Features
+
+- Support for Node.js 8 and IE 11 has been dropped. Please see our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+- Changed TS compilation target to ES2017 to produce smaller bundles and use more native platform features.
+- Updated our internal core package dependencies to their latest versions to add support for Opentelemetry 1.0.0, which is compatible with the latest versions of our other client libraries.
```

Comment:
Wow! Am surprised there are no code changes accompanying this. Is that because search does not have tests for tracing?

Reply:
Most packages do not need code changes at the package level. This is really only an issue because we call `Span#context` in `tracingPolicy` which unfortunately is the method they decided to rename...

Reply:
With that said, there is one build that is failing - so maybe code changes will be coming 😄 ...

I still think no code changes will be needed, as the rest of the builds are passing for this hotfix